### PR TITLE
MovieDetails

### PR DESCRIPTION
**Pull request by MateuszBryczek**
added MovieDetails page showing after clicking in movie tile (MovieDetailsLink in MovieCard).
currently not working as planned. after clicking shows correct link and throw back to movielist. After try to blocking Redirect freeze on loading after clicking.
I guess Navigation or MovieDetails code need fixes.
Error does not affect other elements of the page so further work is safe
I am ready for advices and review

### CODE REVIEW

File: src/features/MovieList/MovieCard/MovieCard.js [hunk 1]
```diff
@@ -30,22 +31,23 @@ const MovieCard = ({
...
-          <TagsWrapper>
-            <GenreList genreIds={genre_ids} />
-          </TagsWrapper>
-          <RateWrapper>
-            <Star src={star} alt="" />
-            <Rating>{vote_average.toFixed(1)}</Rating>
-            <Votes>{vote_count} votes</Votes>
-          </RateWrapper>
-        </DataWrapper>
-      </StyledCard>
```
Comment (gosiagajlusz) [inline src/features/MovieList/MovieCard/MovieCard.js:48]: you deleted  </StyledCard> closing tag, so it appears as error.

File: src/features/PeopleList/PeopleCard/PeopleCard.js [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
     
         return (
         <>
+        <PersonDetailsLink to={`/personDetails/${id}`}>
             <Image src={IMG_URL_SMALL + profile_path}></Image>
-
                 <Name>{name}</Name>
-
+                {personRole && <p>{personRole}</p>}
+            </Description>
```
Comment (gosiagajlusz) [inline src/features/PeopleList/PeopleCard/PeopleCard.js:22]: Opening tag is missing. So maybe  just <Descption/> would be ok

**Review by gosiagajlusz — CHANGES_REQUESTED** [review]
I'm still looking for how error from peoplesaga should be fixed, but for now I wanted to send this, so you can fix it from your branch directly :)

**Review by gosiagajlusz — APPROVED** [review]
now there are less errors ;)

**Review by MateuszBryczek — COMMENTED** [review]
MovieCard fixed but github doesn't show changes yet

**Pull request closed without merging**